.rubocop_todo.yml

```diff
@@ -9,7 +9,8 @@
 # Offense count: 2
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 258
+  # TODO change only for SymphonyClient, or refactor SymphonyClient
+  Max: 280
```

Comment:
I think we can add an exclude for SymphonyClient to https://github.com/psu-libraries/myaccount/blob/main/.rubocop.yml